scripts/hcnmgr

```diff
@@ -691,6 +735,30 @@ done
 #Log this scripts command line to syslog
 hcnlog INFO "$HCNCMD $*"
 
+#Enable bash -x or -vx call trace if requested
+if [ "X$HCNTRACE" != "X" ] ; then
+	set "$HCNTRACE"
+fi
+
+#Validate this tool is running on powerpc platform
+. "$PSERIES_PLATFORM"
+if [ "$platform" != "$PLATFORM_PSERIES_LPAR" ]; then
+	hcnlog INFO "HNV is only supported on PowerVM LPAR"
+	hcnlog INFO "$HCNCMD exit"
+	exit 0
+fi
+
+#Init the distribution and the network service
+check_network_service
+
+#Validate bonding module is loaded
+if ! lsmod | grep -q bonding; then
+	hcnlog DEBUG "HCNMGR: Bonding module not loaded, load module ..."
+	if ! modprobe bonding $BOND_MODPROBE_OPTS; then
+		err $E_NOMODULE
+	fi
+fi
```

Comment:
modprobe should succeed when invoked on an already-loaded module, so the lsmod conditional is unnecessary.

Reply:
okay, will remove that extra check